src/azul/chalice.py

```diff
@@ -150,6 +150,20 @@ def __init__(self,
         self.register_middleware(self._api_gateway_context_middleware, 'http')
         self.register_middleware(self._authentication_middleware, 'http')
 
+    def _add_contact_to_spec(self, spec: JSON) -> JSON:
+        try:
+            description = spec['info'].pop('description')
```

Comment:
pop() is a modification. The argument is declared as JSON (immutable). Please pair with @nadove-ucsc to help you with this.

Reply:
https://ucsc-gi.slack.com/archives/C705Y6G9Z/p1734730433247589